packages/store-ui/src/Auth/Providers/index.tsx

```diff
@@ -22,6 +22,6 @@ export { default as FacebookOAuthButton } from './Facebook/Button'
 // Google
 export { default as GoogleOAuthButton } from './Google/Button'
 // Theme
-export { default as AuthProvidersTheme } from './theme'
+export { default as authProvidersTheme } from './theme'
```

Comment:
Why is it lowercase?

Reply:
Because it's a const, we're using uppercase only for components.

Reply:
oh, I thought it was a component 👌